internal/artifacts/mtad_test.go

```diff
@@ -25,39 +25,23 @@ var _ = Describe("Mtad", func() {
 		Ω(os.RemoveAll(getTestPath("result"))).Should(Succeed())
 	})
 
-	var _ = Describe("ExecuteGenMtad", func() {
+	var _ = Describe("ExecuteMtadGen", func() {
 		It("Sanity", func() {
-			createDirInTmpFolder("mta", "node-js")
-			Ω(ExecuteGenMtad(getTestPath("mta"), getTestPath("result"), nil, "cf", os.Getwd)).Should(Succeed())
+			Ω(ExecuteMtadGen(getTestPath("mta"), getTestPath("result"), nil, "cf", os.Getwd)).Should(Succeed())
 			Ω(getTestPath("result", "mtad.yaml")).Should(BeAnExistingFile())
 		})
-		It("Fails on creating META-INF folder", func() {
-			createDirInTmpFolder("mta")
-			file, err := os.Create(getFullPathInTmpFolder("mta", "META-INF"))
-			Ω(err).Should(Succeed())
-			Ω(file.Close()).Should(Succeed())
-			Ω(ExecuteGenMtad(getTestPath("mta"), getTestPath("result"), nil, "cf", os.Getwd)).Should(HaveOccurred())
-		})
-		It("Fails on location initialization", func() {
-			Ω(ExecuteGenMtad("", getTestPath("result"), nil, "cf", func() (string, error) {
-				return "", errors.New("err")
-			})).Should(HaveOccurred())
-		})
 		It("Fails on platform validation", func() {
-			Ω(ExecuteGenMtad(getTestPath("mta"), getTestPath("result"), nil, "ab", func() (string, error) {
+			Ω(ExecuteMtadGen(getTestPath("mta"), getTestPath("result"), nil, "ab", func() (string, error) {
 				return "", errors.New("err")
 			})).Should(HaveOccurred())
 		})
 		It("Fails on wrong source path - parse fails", func() {
-			Ω(ExecuteGenMtad(getTestPath("mtax"), getTestPath("result"), nil, "cf", os.Getwd)).Should(HaveOccurred())
-		})
-		It("Fails on broken extension file - parse ext fails", func() {
```

Comment:
Return this test - it's supposed to check that generation fails when a bad extension file is sent. It should be fixed though: instead of sending `nil` send `[]string{"cf-mtaext.yaml"}`